src/lobster-bisque.md

```diff
@@ -0,0 +1,50 @@
+# Lobster bisque
+
+Creamy french lobster soup perfect for the hollidays, you can optionally replace the lobster with any other shellfish.
+
+- ⏲️ Prep time: 20
+- 🍳Cook time: 30 min
+- 🍽️ Servings: 2 litres
+
+## Ingredients
+
+- 1 live lobster that weighs ~750 grams
+- 2 onions
+- 2 carrots
+- 1 shallot
+- 2 garlic cloves
+- cognac
+- 3 tbsp concentrated tomato puree
+- 4 tomatoes 
+- 3 tbsp flour
+- 1 glass dry white wine
+- 2 litres fish stock
+- 1 bayleaf
+- 2 twigs of thyme
+- fresh parsley
+- heavy cream
+- canola oil
+- 2 tbsp butter
+
+## Directions
+
+1. Finely dice the onions, carrots, shallots and garlic and set aside for later.
+2. Blanche the lobster in boiling water for 2 minutes, remove from the water and let it leak.
+3. Cut it down the middle and remove the meat from the shell, set aside the meat for later.
+4. Place a stockpot on medium high heat and heat up enough oil to cover the bottom of the pan, add the lobster shells and cook them until they are bright red.
+5. Remove the oil from the pan and discard it, add your butter to the shells.
+6. Add all diced vegetables and let it sauté for 2 minutes, add a dash of cognac and flambé, lower the heat to medium low.
+7. Rougly dice the tomatoes and add them together with the tomato puree, once again let it sauté for 2 minutes.
+8. Add the flour and give it all a stir, let it dry up in the pan.
+9. Deglaze the pan with the white wine and once the wine has dried up a bit add the fish stock.
+10. Let it simmer on low heat for 30 minutes, while the soup is boiling take the lobster meat and dice it into bitesized pieces.
+11. After the soup is done boiling run it through a fine sieve.
+12. Bring the strained soup back to  a light boil and add the diced lobster pieces.
+13. Let the soup boil for 4 more minutes to heat up the lobster and serve, finish of in the plate with a bite of heavy cream.
+
+## Contribution
+
+Yiusa
+-eth '0x68f1317c6512f0267fa711cafb6c134ae968fa80'
```

Comment:
Not sure if tagging your ETH wallet is allowed.

Reply:
I've tagged it on all my recipes i have posted here and they all got added before, if monero and btc are allowed i don't see why eth wouldn't be.

Reply:
Ok. Yeah I didn't read your other PRs closely

Reply:
BTC and XMR addresses are allowed, what's so different about ETH?

Reply:
Eth is obviously fine, but the format should be (in an unordered list):

```
- Yiusa, eth `0x68f1317c6512f0267fa711cafb6c134ae968fa80`
```

I've been going around and fixing yours to make them consistent with others.